app/client/ui/ShareMenu.ts

```diff
@@ -276,7 +276,7 @@ function menuExports(doc: Document, pageModel: DocPageModel) {
       menuItem(() => gristDoc.app.comm.showItemInFolder(doc.name),
         t("Show in folder"), testId('tb-share-option')) :
         menuItem(() => downloadDocModal(doc, pageModel),
-        menuIcon('Download'), t("Download..."), testId('tb-share-option'))
+        menuIcon('Download'), t("Download"), "...", testId('tb-share-option'))
```

Comment:
I change that because the traduction was not tacking in account and it fixed it .. don't know why

Reply:
Arf, I think that's not good for languages written in Right-To-Left (like Arabic or Hebrew).

Reply:
ooooh your right ... I will try again to understand why here it's not working